### PR TITLE
C369085 Invoices | Results List | Verify that value in "Vendor invoice number" column is hyperlink

### PR DESCRIPTION
**Pull request by KazachenkaTatsiana**
…r invoice number" column is hyperlink
https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5915/console
https://foliotest.testrail.io/index.php?/cases/view/369085
image

### CODE REVIEW

File: cypress/support/fragments/invoices/invoiceView.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ const linksAndDocumentsSection = Section({ id: 'documents' });
 // voucher details
 const voucherExportDetailsSection = invoiceDetailsPane.find(Section({ id: 'batchVoucherExport' }));
 const voucherInformationSection = invoiceDetailsPane.find(Section({ id: 'voucher' }));
+const vendorDetailsSection = invoiceDetailsPane.find(Section({ id: 'vendorDetails' }));
```
Comment (manowarqqq) [inline cypress/support/fragments/invoices/invoiceView.js:46]: already exists

File: cypress/support/fragments/invoices/invoiceView.js [hunk 2]
```diff
@@ -281,4 +287,25 @@ export default {
         .click(),
     );
   },
+  verifyInvoicesList() {
+    cy.expect(MultiColumnList({ id: 'invoices-list' }).exists());
+  },
+  verifyInvoicesListIncludeLinkExists(linkName) {
```
Comment (manowarqqq) [inline cypress/support/fragments/invoices/invoiceView.js:293]: verifyInvoicesListIncludeLink() or verifyInvoiceLinkExists

File: cypress/e2e/invoices/vendor-invoice-number-is-hyperlink.cy.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+          path: TopMenu.invoicesPath,
+          waiter: Invoices.waitLoading,
+        });
+      },
+    );
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken();
+    Organizations.deleteOrganizationViaApi(testData.organization.id);
```
Comment (manowarqqq) [inline cypress/e2e/invoices/vendor-invoice-number-is-hyperlink.cy.js:70]: Delete budget, butchgroup

**Review by manowarqqq — COMMENTED** [review]
Build jobs are failed

**Review by manowarqqq — APPROVED** [review]

File: cypress/support/fragments/invoices/invoiceView.js [hunk 4]
```diff
@@ -281,7 +284,27 @@ export default {
...
+  selectInvoiceLineByName(linkName) {
+    cy.do(
+      MultiColumnList({ id: 'invoices-list' })
+        .find(MultiColumnListCell({ content: linkName }))
+        .find(Link())
+        .click(),
+    );
 
+    return InvoiceLineDetails;
+},
```
Comment (IhorBohdan) [inline cypress/support/fragments/invoices/invoiceView.js:307]: Fix linter issues

**Review by IhorBohdan — APPROVED** [review]

**Review by manowarqqq — APPROVED** [review]

**Pull request merged**